common/scala/src/main/scala/org/apache/openwhisk/core/connector/Message.scala

```diff
@@ -426,3 +426,22 @@ object EventMessage extends DefaultJsonProtocol {
 
   def parse(msg: String) = Try(format.read(msg.parseJson))
 }
+
+case class ByteSizeMessage(userMemory: ByteSize) extends Message {
```

Comment:
I think the name should denote its purpose more.

Reply:
hm.. already changed it from `ByteSizeMessage` to `UserMemoryMessage`